quickstart/install-terraform.py

```diff
@@ -3,30 +3,33 @@
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
 
+import argparse
 import os
+import pathlib
 import shutil
 import sys
 import urllib.request
 import zipfile
 
+
 TEMP_DIR           = '/tmp'
 TERRAFORM_VERSION  = '0.12.3'
-TERRAFORM_BIN_DIR  = '/usr/local/bin'
-TERRAFORM_BIN_PATH = TERRAFORM_BIN_DIR + '/terraform'
 
-def terraform_install(version=TERRAFORM_VERSION):
+
+def terraform_install(terraform_bin_dir, version=TERRAFORM_VERSION):
```

Comment:
the function would make more sense if "version" is optional.  If it's not specified, install latest.  otherwise, install the specific version.  but let's leave this for a separate PR.